src/mono/mono/mini/mini-amd64.c

```diff
@@ -8404,15 +8400,11 @@ mono_arch_emit_epilog (MonoCompile *cfg)
 	/* Mark the start of the epilog */
 	mono_emit_unwind_op_mark_loc (cfg, code, 0);
 
-#ifdef MONO_ARCH_HAVE_SWIFTCALL
-	if (mono_method_signature_has_ext_callconv (cfg->method->signature, MONO_EXT_CALLCONV_SWIFTCALL)) {
-		if (cfg->arch.swift_error_var && (cfg->arch.swift_error_var->flags & MONO_INST_IS_DEAD)) {
-			MonoInst *ins = cfg->arch.swift_error_var;
-			amd64_mov_reg_membase (code, AMD64_R11, ins->inst_basereg, ins->inst_offset, sizeof (target_mgreg_t));
-			amd64_mov_membase_reg (code, AMD64_R11, 0, AMD64_R12, sizeof (target_mgreg_t));
-		}
+	if (cfg->arch.swift_error_var && (cfg->arch.swift_error_var->flags & MONO_INST_IS_DEAD)) {
+		MonoInst *ins = cfg->arch.swift_error_var;
```

Comment:
Don't think the IS_DEAD part is needed, also if its needed, shouldn't it be !IS_DEAD ?

Reply:
Not needed, updated to use the existing arg ins.